src/main/java/mate/academy/spring/config/WebConfig.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.view.InternalResourceViewResolver;
+import org.springframework.web.servlet.view.JstlView;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = {"mate.academy.spring.controllers"})
+public class WebConfig {
+
+    @Bean
+    public InternalResourceViewResolver resolver() {
```

Comment:
```suggestion
    public InternalResourceViewResolver getResolver() {
```
назва методу - дієслово або фраза на його основі(get, put, createNewFile)